webapp/src/errorList.tsx

```diff
@@ -77,15 +75,15 @@ export class ErrorList extends React.Component<ErrorListProps, ErrorListState> {
 
         return (
             <div className={`errorList ${isCollapsed ? 'errorListSummary' : ''}`} hidden={!errorsAvailable}>
-                <div className="errorListHeader" role="button" onClick={this.onCollapseClick}>
+                <div className="errorListHeader" role="button" onClick={this.onCollapseClick} onKeyDown={sui.fireClickOnEnter} tabIndex={0}>
```

Comment:
Oh woops almost forgot, these need aria labels to indicate what they do for screen readers; each div just needs a short description of what it'll do

```suggestion
                <div className="errorListHeader" role="button" aria-label={lf("{0} error list", isCollapsed ? lf("Expand") : lf("Collapse"))} onClick={this.onCollapseClick} onKeyDown={sui.fireClickOnEnter} tabIndex={0}>
```

(note that I just typed that out in the github comment window, sorry if there's a typo ~)

Reply:
Thanks I'll add those on and leave the outline on for the errors